README.md

```diff
@@ -1 +1,73 @@
-# Helo
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#Hello Microverse)
+  - [🛠 Built With](Html, css)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+
+**[Hello-Microverse]** is a simple demo for using version flow control.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.mdn.com/html/">Html</a></li>
+  </ul>
+</details>
+
+<details>
+  <ul>
+    <li><a href="https://mdn.com/css">Css</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[Git-ignore]**
+- **[Html-header]**
+- **[]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
```

Comment:
- Kindly make sure to have all the missing sections of the README template considering the FAQ as the only optional section. :recycle: please remember to customize them too